release/preview/ubuntu18.04/docker/pwsh-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
```

Comment:
It doesn't look like this file is used?

Reply:
This causes the script to quite if there is an error

Reply:
Right, but I'm saying the entire shell file

Reply:
Fixed